lib/node_modules/@stdlib/blas/base/sscal/benchmark/fortran/Makefile

```diff
@@ -0,0 +1,115 @@
+#/
```

Comment:
This is an "older" version of this file. Use [`ddot`](https://github.com/stdlib-js/stdlib/blob/develop/lib/node_modules/%40stdlib/blas/base/ddot/benchmark/fortran/Makefile) as a reference.